qa/tasks/cephfs/filesystem.py

```diff
@@ -389,7 +389,10 @@ def mds_signal(self, mds_id, sig, silent=False):
     def mds_is_running(self, mds_id):
         return self.mds_daemons[mds_id].running()
 
-    def newfs(self, name='cephfs', create=True):
+    def newfs(self, name='testcephfs_abcd', create=True):
+        if name == 'testcephfs_abcd':
+            import random
+            name = 'testcephfs_' + str(random.randint(1000,  9999))
```

Comment:
I may be missing something but why not:

```suggestion
    def newfs(self, name=None, create=True):
        if name is None:
            import random
            name = f"testcephfs_{''.join(random.choices(string.ascii_uppercase + string.digits, k=16))}"
```

Reply:
Incorporated partially since `testcephfs_3462` is easier to type, find and speak compared to `testcephfs_EMZ70TBOAAIYESQY`. Latter is unnecessarily long and complex.